packages/react/src/components/DataGrid/DataGrid.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import {withDesign} from '../../../.storybook/utils.ts';
+import DataGrid from './DataGrid.tsx';
+
+export const meta = {
+  component: DataGrid,
+  title: StoryConfig.DataGrid.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => <DataGrid {...args} />;
+
+# DataGrid
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+Use the DataGrid component to display information in a grid-like format of rows and columns.
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      columns: [
+        {field: 'id', headerName: 'ID', width: 90},
+        {
+          field: 'firstName',
+          headerName: 'First name',
+          width: 150,
+          editable: true,
+        },
+        {
+          field: 'lastName',
+          headerName: 'Last name',
+          width: 150,
+          editable: true,
+        },
+        {
+          field: 'age',
+          headerName: 'Age',
+          type: 'number',
+          width: 110,
+          editable: true,
+        },
+      ],
+      rows: [
+        {id: 1, lastName: 'Snow', firstName: 'Jon', age: 35},
+        {id: 2, lastName: 'Lannister', firstName: 'Cersei', age: 42},
+        {id: 3, lastName: 'Clifford', firstName: 'Ferrara', age: 44},
+        {id: 4, lastName: 'Frances', firstName: 'Rossini', age: 36},
+        {id: 5, lastName: 'Roxie', firstName: 'Harvey', age: 65},
+      ],
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable story="Overview" />
+
+## Usage
+
+Import and use the `DataGrid` component in your components as follows.
+
+<Source language="jsx" dark format code={dedent`import DataGrid from '@oxygen-ui/react/DataGrid';\n`} />
```

Comment:
Let's add a demo function too to show the usage with an example in the Story.

Refer: https://github.com/wso2/oxygen-ui/blob/main/packages/react/src/components/MenuItem/MenuItem.stories.mdx#L52

Reply:
Fixed